pkgs/development/python-modules/matscipy/default.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  pythonOlder,
+  setuptools,
```

in `pkgs/development/python-modules` you need to import each dependency at a package:
```suggestion
  setuptools,
  meson,
  meson-python,
  ninja,
  cython,
  numpy,
  scipy,
  ase,
  packaging,
```